apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/CreateOrder.tsx

```diff
@@ -115,7 +115,7 @@ export const CreateOrder = () => {
       buy: amounts.buy,
       pools: poolList,
       selectedPool: createOrder.selectedPool,
-      slippage: createOrder.slippage,
+      slippage: createOrder.slippage * 100,
       address: addresses.used[0],
```

Comment:
we we want to always send the same address I'd say we should go for `wallet.externalChain[0]`, you don't necessarily consume the addresses in order, the first used can be the index 1/2/3, we don't know.

Reply:
I can't find the `externalChain ` type in `YoroiWallet` 🤔 should we go with `wallet.externalAddresses[0]` ?

Reply:
wow sorry, yap, externalAddress[0] (here index accessing is safe)

Reply:
still `const [firstExtAddress, ] = wallet.externalAddresses`